css_tasks/task11/scss/_decoration.scss

```diff
@@ -0,0 +1,147 @@
+@import "variables";
+@import "mixins";
+
+.decoration {
+  width: 100%;
+  margin-top: $standardTopPadding;
+}
+.decoration-green {
+  background: $mainThemeColor;
+  span {
+    @include whiteBoldText;
+    font-size: $bigFontSize;
+    display: block;
+    margin: 0 auto;
+  }
+  p {
+    @include normalWhiteText;
+    text-transform: uppercase;
+    text-align: center;
+  }
+}
+.decoration--map {
+  position: relative;
+  background: url("../assets/layer-59@2x.jpg") no-repeat;
+  background-size: cover;
+  background-position: center;
+  &:after {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    background: #d9d9d9;
+    opacity: 0.8;
+    content: "";
+  }
+}
+.decoration__wrapper {
+  display: flex;
+  justify-content: space-between;
+  height: 10rem;
+}
+.decoration__wrapper--carousel {
+  position: relative;
+  height: 100%;
+  align-items: center;
+  img {
+    width: 100%;
+  }
+  .decoration-image__carousel {
+    clip-path: circle(50% at 50% 50%);
+  }
+  &:before {
+    position: absolute;
+    top: 30%;
+    left: 10%;
+    font-size: $bigFontSize;
+    content: "\02c2";
+    cursor: pointer;
+  }
+  &:after {
+    position: absolute;
+    top: 30%;
+    right: 10%;
+    font-size: $bigFontSize;
+    content: "\02c3";
+    cursor: pointer;
+  }
+}
+.decoration__element {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-grow: 1;
+  flex-basis: 15%;
+  height: 100%;
+  border-right: 1px solid $mainBackgroundColor;
+  &:first-child {
+    border-left: 1px solid $mainBackgroundColor;
+    span {
+      margin-top: 1rem;
+    }
+  }
+}
+.decoration-carousel {
+  padding: 5rem 0;
+  background: $decorationBackground;
+  span {
+    padding: 2rem;
+    border: 3px solid $mainThemeColor;
+  }
+  &__text {
+    position: relative;
+    align-self: end;
+    padding-left: 2rem;
+    @include subTitle;
+    font-size: $carouselText;
+    text-align: left;
+
+    &:before {
```

Comment:
some reminder to use `::` for pseudo elements

Reply:
ok